Vagrantfile

```diff
@@ -129,9 +129,11 @@ module Forklift
   plugin_vagrantfiles.each { |f| load f }
   plugin_base_boxes
   unsorted = @box_loader.add_boxes('boxes.yaml', 'config/versions.yaml') if File.exists?('boxes.yaml')
-  @boxes  = unsorted.keys.sort.inject({}) do |hash, key|
-    hash[key] = unsorted[key]
-    hash
+  if unsorted
+    @boxes  = unsorted.keys.sort.inject({}) do |hash, key|
```

Comment:
I think it's needed because it's adding and sorting the boxes that are taken from the boxes.yaml file, and then sorts them.